NEWS.md

```diff
@@ -1,4 +1,7 @@
 # devtools 1.12.0.9000
+* Added release_check for files in inst/doc/ that could prevent 
```

Comment:
Missing new line at start. `inst/doc` should be in quotes. Second line should be indented.